src/client/eppo-client.ts

```diff
@@ -23,42 +16,21 @@ import {
   MAX_EVENT_QUEUE_SIZE,
   POLL_INTERVAL_MS,
 } from '../constants';
-import { IAllocation } from '../dto/allocation-dto';
-import { IExperimentConfiguration } from '../dto/experiment-configuration-dto';
-import { IVariation } from '../dto/variation-dto';
-import { EppoValue, ValueType } from '../eppo_value';
-import ExperimentConfigurationRequestor from '../experiment-configuration-requestor';
+import { EppoValue } from '../eppo_value';
+import { Evaluator, FlagEvaluation, noneResult } from '../evaluator';
+import ExperimentConfigurationRequestor from '../flag-configuration-requestor';
 import HttpClient from '../http-client';
+import { Flag, VariationType } from '../interfaces';
 import { getMD5Hash } from '../obfuscation';
 import initPoller, { IPoller } from '../poller';
-import { findMatchingRule } from '../rule_evaluator';
-import { getShard, isShardInRange } from '../shard';
+import { AttributeType } from '../types';
 import { validateNotBlank } from '../validation';
 
 /**
  * Client for assigning experiment variations.
  * @public
  */
 export interface IEppoClient {
```

Comment:
If easy, now would be a good time to ax `IEppoClient`
(It's probably not easy)

Reply:
Why do you want to axe it?

Reply:
Because it's a huge pain in the butt to have to define the signature in two places 😛